base/screens/menu.js

```diff
@@ -83,7 +83,7 @@ export class MenuScreen {
 
 export function initialize_menu(game, data) {
     data.spacebar_input.add(() => {
-        if (data.hero.in_action() || data.in_battle || !data.created) return;
+        if (data.hero.in_action() || data.in_battle || !data.created || data.in_dialog) return;
```

Comment:
GoldenSun.in_dialog does not exist anymore and that's indeed a bug. I think that the appropriate flag here would be `data.game_event_manager.on_event`.